RELEASING.md

```diff
@@ -41,9 +41,8 @@ If review is not required, commit directly to `dev`.
    * Close the repository and wait for it to verify.
    * Release it.
    
-5. Update documentation website:<br>
-    * Set new value for [`kotlinx.serialization.release.tag`](https://buildserver.labs.intellij.net/admin/editProject.html?projectId=Kotlin_KotlinSites_Builds_KotlinlangOrg_LibrariesAPIs&tab=projectParams)
-    * And run deploy [configuration](https://buildserver.labs.intellij.net/buildConfiguration/Kotlin_KotlinSites_Builds_KotlinlangOrg_KotlinSerializationApi?branch=%3Cdefault%3E&mode=builds)
+5. Propose the website documentation update:<br>
+    * Set new value for [`KOTLINX_SERIALIZATION_RELEASE_TAG`](https://github.com/JetBrains/kotlin-web-site/blob/master/.teamcity/BuildParams.kt), creating a Pull Request in the website's repository.
```

Comment:
It's not clear what happens after the PR creation. Should releaser wait for PR to be merged or website build is started right away? If the latter, can PR be closed without merging then?

Reply:
Thanks for pointing it out. Please, review the updates.